tekore/client/process.py

```diff
@@ -24,3 +24,12 @@ def model_list(type_: type, top_item: str):
     def post_func(json: dict):
         return ModelList(type_(**i) for i in json[top_item])
     return post_func
+
+def cursor_paging(type_: type, top_item: str):
```

Comment:
This name can be changed to something more generic. Right now it is used for parsing into `FullArtistCursorPaging` object in `follow.followed_artists`. Up for a discussion

Reply:
Perhaps I would change it. It isn't really tied to the idea of paging as much as accessing an item and using that to unpack, as you already mentioned in the docstring.

Reply:
In new commit I modified `single` to accept `top_item`. I think this way is better, what do you think?

Reply:
I agree!